dbt/adapters/redshift/relation_configs/materialized_view.py

```diff
@@ -182,10 +182,14 @@ def parse_relation_results(cls, relation_results: RelationResults) -> dict:
             "mv_name": materialized_view.get("table"),
             "schema_name": materialized_view.get("schema"),
             "database_name": materialized_view.get("database"),
-            "autorefresh": materialized_view.get("autorefresh"),
             "query": cls._parse_query(query.get("definition")),
         }
 
+        autorefresh_value = materialized_view.get("autorefresh")
```

Comment:
Redshift returns the value as a single character: `'t'` or `'f'`. Sometimes `agate` picks this up and converts and sometimes it does not.